tests/brokers/nats/test_consume.py

```diff
@@ -248,3 +248,56 @@ async def handler(msg: NatsMessage):
             m.mock.assert_not_called()
 
         assert event.is_set()
+
+    @pytest.mark.asyncio()
+    async def test_consume_kv(
+        self, queue: str, full_broker: NatsBroker, event: asyncio.Event
+    ):
+        @full_broker.subscriber("hello", kv_watch="test")
+        async def handler(filename: str):
```

Comment:
KV subscriber consumes not just a filename, but serialized body. Please, test it by mocks.